docs/source/multidc/multidc.md

```diff
@@ -0,0 +1,455 @@
+                operator: In
+                values:
+                - scylla
+        tolerations:
+        - key: role
+          operator: Equal
+          value: scylla-clusters
+          effect: NoSchedule
+    - name: c
+      members: 1
+      storage:
+        storageClassName: scylladb-local-xfs
+        capacity: 1800G
+      resources:
+        limits:
+          cpu: 7
+          memory: 56G
+      placement:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: topology.kubernetes.io/zone
+                operator: In
+                values:
+                - us-east-1c
+              - key: scylla.scylladb.com/node-type
+                operator: In
+                values:
+                - scylla
+        tolerations:
+        - key: role
+          operator: Equal
+          value: scylla-clusters
+          effect: NoSchedule
+```
+
+Note, that
+
+Apply the manifest:
+```shell
+kubectl --context="${CONTEXT_DC1}" apply --server-side -f=dc1.yaml
+```
+
+Wait for the cluster to be fully rolled out:
+```shell
+kubectl --context="${CONTEXT_DC1}" -n=scylla wait --for='condition=Progressing=False' scyllaclusters.scylla.scylladb.com/scylla-cluster
+```
+```console
+scyllacluster.scylla.scylladb.com/scylla-cluster condition met
+```
+
+```shell
+kubectl --context="${CONTEXT_DC1}" -n=scylla wait --for='condition=Degraded=False' scyllaclusters.scylla.scylladb.com/scylla-cluster
+```
+```console
+scyllacluster.scylla.scylladb.com/scylla-cluster condition met
+```
+
+```shell
+kubectl --context="${CONTEXT_DC1}" -n=scylla wait --for='condition=Available=True' scyllaclusters.scylla.scylladb.com/scylla-cluster
+```
+```console
+scyllacluster.scylla.scylladb.com/scylla-cluster condition met
+```
+
+You can now verify that all the nodes of your cluster are in UN state:
+```shell
+kubectl --context="${CONTEXT_DC1}" -n=scylla exec -it pod/scylla-cluster-us-east-1-a-0 -c=scylla -- nodetool status
+```
+
+The expected output should look similar to the below:
+```console
+Datacenter: us-east-1
+=====================
+Status=Up/Down
+|/ State=Normal/Leaving/Joining/Moving
+--  Address      Load       Tokens       Owns    Host ID                               Rack
+UN  10.0.70.195  290 KB     256          ?       494277b9-121c-4af9-bd63-3d0a7b9305f7  c
+UN  10.0.59.24   559 KB     256          ?       a3a98e08-0dfd-4a25-a96a-c5ab2f47eb37  b
+UN  10.0.19.237  107 KB     256          ?       64b6292a-327f-4128-852a-6004039f402e  a
+```
+
+##### Retrieve PodIPs of ScyllaDB nodes for use as external seeds
+
+``` warning::
+    Due to the ephemeral nature of PodIPs, it is ill-advised to use them as seeds in production environments.
+    PodIPs are being used in this example for the sheer simplicity of this setup.
+    In production environments, it is recommended that you use domain names or non-ephemeral IP addresses.
```

Comment:
```suggestion
    In production environments, it is recommended that you use domain names.
```

Sounds like we discourage from using PodIPs, but we don't.

Reply:
we do for external seeds though, don't we?

Reply:
right it's mentioned in seed context. 
Anyway, this warning could use describing what's the problem of using ephemeral ones, which is that they in future may no longer point to nodes from this ScyllaCluster. To solve it, we recommend setting up DNS which would resolve to ephemeral IPs, or somehow provide ScyllaCluster Pods with static IP address and use them as seeds.

Reply:
done